tests/test_global_challenge.py

```diff
@@ -39,7 +41,30 @@ def setUp(self):
         test_user.inhabitants = 2
         test_user.set_password('some_password')
         test_user.state = StateType.ACTIVE
+        test_first_timestamp = datetime(2020, 2, 13, 9, 57, 3)
+        test_first_user_saving = UserSaving(test_first_timestamp,
```

Comment:
Maybe use different values for first and second savings, as well as first and second base lines?